examples/bert/run_pretraining.py

```diff
@@ -348,19 +390,28 @@ def main(_):
     model(model.inputs)
     model.summary()
 
+    #Implemented Learning Rate Schedule
+    learning_rate_schedule = LinearDecayWithWarmup(
+        learning_rate = FLAGS.learning_rate,
+        num_warmup_steps = FLAGS.num_warmup_steps,
+        num_train_steps = FLAGS.num_train_steps)
+    
     # Wrap with pretraining heads and call fit.
     pretraining_model = BertPretrainer(model)
     pretraining_model.compile(
         # TODO(mattdangerw): Add AdamW and a learning rate schedule.
```

Comment:
You could remove the learning rate schedule bit from this TODO.

Reply:
Ah, yes. Alright!